var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -54,3 +57,5 @@ def configure_args(self):
             '--enable-pc-files',
             '--with-pkg-config-libdir={0}/lib/pkgconfig'.format(self.prefix)
         ]
+        if '+symlinks' in self.spec:
+            opts += ["--enable-symlinks"]
```

Comment:
It looks like this happened during @tgamblin's develop merge. Can you add `return opts` to the end?